src/se/vidstige/jadb/AdbServerLauncher.java

```diff
@@ -18,17 +18,23 @@ public AdbServerLauncher(Runtime runtime) {
 
     private String findAdbExecutable() {
         String android_home = System.getenv("ANDROID_HOME");
-        if (android_home == null || android_home.equals("")) {
-            return "adb";
-        }
-
-        return android_home + "/platform-tools/adb";
+        return android_home != null && !android_home.equals("")?android_home + "/platform-tools/adb":"adb";
```

Comment:
I feel this one liner hurt readability, please use the previous syntax for clarity.